### ISSUE TITLE
AttributeError: 'Recognizer' object has no attribute 'recognize_houndify'

### ISSUE THREAD

**Issue by surendrasalkekore** [Uberi/speech_recognition#154]
## Steps to reproduce

```
print("Houndify thinks you said " + r.recognize_houndify(audio, client_id=HOUNDIFY_CLIENT_ID, client_key=HOUNDIFY_CLIENT_KEY))
```

AttributeError: 'Recognizer' object has no attribute 'recognize_houndify'
## Expected behaviour

(
    print("Houndify thinks you said " + r.recognize_houndify(audio, client_id=HOUNDIFY_CLIENT_ID, client_key=HOUNDIFY_CLIENT_KEY))
AttributeError: 'Recognizer' object has no attribute 'recognize_houndify'

![recognize_houndify](https://cloud.githubusercontent.com/assets/20574324/18088137/8555d5f4-6ed6-11e6-883c-70edbf7897b1.png)

**Comment by jhoelzl** [comment]
Houndify support is not officially released now. Which version of this library do you use? Have a look on https://github.com/Uberi/speech_recognition/releases, you will see that version 3.4.6 does not include Houndify, but there are 12 new commits (https://github.com/Uberi/speech_recognition/compare/3.4.6...master). 

Download the master branch (and do not install the module with `pip`, then your error should be solved.

**Comment by Uberi** [comment]
Fixed in 3.5.0!
